plugins/sh/src/com/intellij/sh/ShSupport.java

```diff
@@ -19,7 +21,12 @@
 import org.jetbrains.annotations.Nullable;
 
 public interface ShSupport {
-  static ShSupport getInstance() { return ServiceManager.getService(ShSupport.class); }
+  ExtensionPointName<ShSupport> EP_NAME = ExtensionPointName.create("com.jetbrains.sh.shSupport");
```

Comment:
Let's make it `public static final`

Reply:
Isn't this already `public static final` because it's in an interface?